test/e2e/tls_test.go

```diff
@@ -131,27 +134,30 @@ func (data *TestData) opensslConnect(t *testing.T, pod string, container string,
 	}{
 		{
 			clusterInfo.podV4NetworkCIDR != "",
-			"127.0.0.1",
+			nodeIPv4,
 			"-4",
 		},
 		{
 			clusterInfo.podV6NetworkCIDR != "",
```

Comment:
you probably no longer need a dedicated `enabled` boolean to decide whether you should run the test. You can just use `c.ip == ""` as the test to decide whether the case should skipped.